extension/chrome/elements/compose.htm

```diff
@@ -126,7 +126,7 @@ <h1 id="header_title" data-test="header-title">New Secure Message</h1>
             (messages sent to FlowCrypt users have no limit, and don't need password exchange)
           </div>
           <div>
-            <div class="warning_nopgp" data-test="warning-nopgp">emails <span class="grey">in grey</span> don't use encryption </div>
+            <div class="warning_nopgp" data-test="warning-nopgp">emails <span class="grey">in grey</span> don't use encryption, </div>
```

Comment:
How would it look if we have `warning_nopgp` visible and `warning_revoked` non-visible? The comma will be the terminating character in this situation. What is the simplest way to show the comma only if both `warning_nopgp` and `warning_revoked` are set? Can we do this with CSS or do we need to edit this code block?
```
    this.view.S.cached('warning_nopgp').css('display', anyNopgp ? 'inline-block' : 'none');
    this.view.S.cached('warning_revoked').css('display', anyRevoked ? 'inline-block' : 'none');
```

Reply:
> How would it look if we have warning_nopgp visible and warning_revoked non-visible?

In which situation it's the case? As I understand correctly both are shown always, even if there are no orange recipients:

![image](https://user-images.githubusercontent.com/6059356/120822506-24820d80-c55f-11eb-9fd2-59086322f48a.png)

Reply:
BTW @rrrooommmaaa I recommend installing the `htmlhint` extension which will help to prevent mistakes in HTML.

Right now, the markup has issues:

![image](https://user-images.githubusercontent.com/6059356/120823064-b0943500-c55f-11eb-9732-87318b63bffc.png)

Reply:
> In which situation it's the case? As I understand correctly both are always shown, even if there are no orange recipients

@limonte This is probably a bug, as the `warning_nopgp` is supposed to be shown if at least on recipient has NO_PGP status, whereas `warning_revoked` should be shown if at least one recipient has `REVOKED` status, as is implemented by this piece of code:
```
await this.showMsgPwdUiAndColorBtn(
        this.view.recipientsModule.getRecipients().some(r => r.status === RecipientStatus.NO_PGP),
        this.view.recipientsModule.getRecipients().some(r => r.status === RecipientStatus.REVOKED),
      )
```
And the comma is supposed to be shown only if both checks are true.

> There are two blocks with .warning_nopgp class, that should be fixed I believe

Yes, there should be two separate blocks turned on by `NO_PGP` as per this discussion
https://github.com/FlowCrypt/flowcrypt-browser/pull/3658#discussion_r636911000_

Reply:
> This is probably a bug, as the warning_nopgp is supposed to be shown if at least on recipient has NO_PGP status, whereas warning_revoked should be shown if at least one recipient has REVOKED status

Reported the issue: #3726 

I removed my commit with the comma from this PR, let's handle that in #3726 as well.